spark/src/main/scala/org/apache/comet/serde/QueryPlanSerde.scala

```diff
@@ -2442,10 +2442,8 @@ object QueryPlanSerde extends Logging with ShimQueryPlanSerde with CometExprShim
           return None
         }
 
-        if (join.buildSide == BuildRight) {
-          // DataFusion HashJoin assumes build side is always left.
-          // TODO: support BuildRight
-          withInfo(join, "BuildRight is not supported")
+        if (join.buildSide == BuildRight && join.joinType == LeftAnti) {
+          withInfo(join, "BuildRight with LeftAnti is not supported")
```

Comment:
Do we have a tracking issue yet for supporting BuildRight + LeftAnti?

Reply:
There is one #457.